streakerbar/AppDelegate.swift

```diff
@@ -70,7 +80,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         let data: NSData? = NSData(contentsOfFile: location)
         
         if let fileData = data {
-            let content = NSString(data: fileData, encoding:NSUTF8StringEncoding) as String
+            let content = NSString(data: fileData, encoding:NSUTF8StringEncoding) as! String
```

Comment:
compile errors around this line:

![screen shot 2015-03-28 at 2 46 49 pm](https://cloud.githubusercontent.com/assets/7292/6882385/4caeaed2-d559-11e4-8d06-a7f55dcb0281.png)